dev_guide/templates.adoc

```diff
@@ -51,8 +51,16 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
+== Creating a Project From a Template
```

Comment:
this whole section has a confusing flow.  for starters you don't create projects from templates, projects are containers for "stuff" and templates create some stuff.  Some stuff in a project might be created from a template, some might be created in other ways.

secondly, there are a few conflated flows here:
1) loading a template into the openshift template library so it can be instantiated ("osc create -f mytemplate.json" is the only way to do this today as far as i know)
2) instantiating a template (choose the template via the webconsole, or run "osc process mytemplate | osc create -f -"
3) you can also instantiate a template that's not in the library (ie directly from the filesystem) by running "osc process -f mytemplate.json | osc create -f -"
4) you will also soon be able to instantiate templates via Eclipse, so we wil want at least a call out to docs for that.. @jcantrill can help there.